cmd/trcli/config/config_test.go

```diff
@@ -9,10 +9,8 @@ import (
 )
 
 func TestParseTransferYaml_WithEnvSubstitution(t *testing.T) {
-	require.NoError(t, os.Setenv("FOO", "secret1"))
-	require.NoError(t, os.Setenv("BAR", "secret2"))
-	defer os.Unsetenv("FOO")
-	defer os.Unsetenv("BAR")
+	t.Setenv("FOO", "secret1")
```

Comment:
this change better to extract in separate PR (replacing os.Setenv with t.Setenv).